model/integration_action.go

```diff
@@ -157,14 +157,17 @@ type PostActionIntegration struct {
 }
 
 type PostActionIntegrationRequest struct {
-	UserId     string                 `json:"user_id"`
-	ChannelId  string                 `json:"channel_id"`
-	TeamId     string                 `json:"team_id"`
-	PostId     string                 `json:"post_id"`
-	TriggerId  string                 `json:"trigger_id"`
-	Type       string                 `json:"type"`
-	DataSource string                 `json:"data_source"`
-	Context    map[string]interface{} `json:"context,omitempty"`
+	UserId      string                 `json:"user_id"`
+	UserName    string                 `json:"user_name"`
+	ChannelId   string                 `json:"channel_id"`
+	ChannelName string                 `json:"channel_name"`
+	TeamId      string                 `json:"team_id"`
+	TeamDomain  string                 `json:"team_domain"`
```

Comment:
Please use `TeamName` here. Even though it's not compliant with slack, it follow the Mattermost naming convention.

Reply:
No problem, I'll make the change.
FYI, I used TeamDomain because it's used in outgoing_webhook.go
https://github.com/mattermost/mattermost-server/blob/e62471dff6082c4e3efb37ea27125ee182083980/model/outgoing_webhook.go#L38